lib/openssl.cpp

```diff
@@ -104,46 +104,16 @@ Value OpenSSL_KDF_pbkdf2_hmac(Env *env, Value self, Args args, Block *) {
     args.ensure_argc_is(env, 1);
     auto pass = args.at(0)->to_str(env);
     if (!kwargs) kwargs = new HashObject {};
-    TM::Vector<TM::String> missing_keywords {};
-    auto salt = kwargs->remove(env, "salt"_s);
-    if (!salt) {
-        missing_keywords.push("salt");
-    } else {
-        salt = salt->to_str(env);
-    }
-    auto iterations = kwargs->remove(env, "iterations"_s);
-    if (!iterations) {
-        missing_keywords.push("iterations");
-    } else {
-        iterations = iterations->to_int(env);
-    }
-    auto length = kwargs->remove(env, "length"_s);
-    if (!length) {
-        missing_keywords.push("length");
-    } else {
-        length = length->to_int(env);
-    }
+    env->ensure_no_missing_keywords(kwargs, { "salt", "iterations", "length", "hash" });
```

Comment:
I totally forgot about `ensure_no_missing_keywords`. We should probably move that to the `Args` class to match the other `ensure_*` methods...